recipes/geosnap/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "geosnap" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/spatialucr/geosnap.git
```

Comment:
Don't use `git` but the tarballs github provides or the PyPI package: https://pypi.org/project/geosnap/

Reply:
will do, thanks. I'll make the same change in [giddy](https://github.com/conda-forge/staged-recipes/pull/8230) as well